packages/tooling/.eslintrc.json

```
@@ -5,7 +5,8 @@
     "eslint:recommended",
     "plugin:@typescript-eslint/recommended",
     "prettier",
-    "plugin:react/recommended"
+    "plugin:react/recommended",
+    "plugin:cypress/recommended"
```

```suggestion
    "plugin:react/recommended"
```

let's remove it, it's messing with the code, and it doesn't seem neccessary